include/variables.h

```diff
@@ -2932,7 +2932,6 @@ extern GameInfo* gGameInfo; // 0x8015FA90
 extern u16 D_8015FCC0;
 extern u16 D_8015FCC2;
 extern u16 D_8015FCC4;
-//extern ? D_8015FCC6;
 extern u8 D_8015FCC8;
 //extern ? D_8015FCCA;
```

Comment:
can probably remove the other too?

Reply:
its local to z_demo though, only global things need to go here

Reply:
`D_8015FCC0` Used in treemouth.c
`D_8015FCC2` Used in treemouth.c
`D_8015FCC4` Used in treemouth.c
`D_8015FCC8` Used in dbcamera.c and kankyo.c

Reply:
I was talking about D_8015FCCA, by "other" i meant the other which is commented out

Reply:
Ok I wasn't sure.  I took it out anyway but I guess it came back.